Gemfile.lock

```diff
@@ -76,16 +76,14 @@ GEM
       momentjs-rails (~> 2.8)
       sass-rails (~> 5.0)
       selectize-rails (~> 0.6)
-    api-pagination (4.7.1)
+    api-pagination (4.7.0)
```

Comment:
Running `docker-compose run web bundle update` on the current commit of this branch results in the Gemfile.lock being nearly identical to the one on master.  There are a few gems that have received minor updates since the last round of updates.

Reply:
Huh. I can replicate that too. I have no idea how I got into this situation then, I never explicitly downgraded any gems.

Whatever, I'll commit the new lockfile and leave this to the bundle gods.

Reply:
:pray: Bundle Gods :pray: